enterprise/internal/campaigns/state.go

```diff
@@ -108,15 +108,15 @@ func ComputeCheckState(c *campaigns.Changeset, events ChangesetEvents) campaigns
 	return campaigns.ChangesetCheckStateUnknown
 }
 
-// ComputeChangesetState computes the overall state for the changeset and its
+// ComputeChangesetExternalState computes the overall state for the changeset and its
 // associated events. The events should be presorted.
-func ComputeChangesetState(c *campaigns.Changeset, history []changesetStatesAtTime) (campaigns.ChangesetState, error) {
+func ComputeChangesetExternalState(c *campaigns.Changeset, history []changesetStatesAtTime) (campaigns.ChangesetExternalState, error) {
```

Comment:
I think `ComputeExternalState` is the better name. This one here's quite the mouthful 😄 

```suggestion
func ComputeExternalState(c *campaigns.Changeset, history []changesetStatesAtTime) (campaigns.ChangesetExternalState, error) {
```